charts/argo-events/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: v1.7.6
+appVersion: v1.8.0
 description: A Helm chart for Argo Events, the event-driven workflow automation framework
 name: argo-events
-version: 2.3.3
+version: 2.3.4
```

Comment:
This PR updates version of argo-events and add feature, so I think bumping minor version to `2.4.0` would be better? 🤔

Reply:
It does make sense. Done.